packages/scheduler-prisma/src/helpers.ts

```diff
@@ -327,7 +327,7 @@ export const timestampHelperFactory =
 
     if (!jobInstance) {
       try {
-        return task(args, context);
+        return await task(args, context);
```

Comment:
With the await, the `try/catch` won't actually catch any errors

Reply:
I didn't get it. Did you mean `Without` the await exceptions won't be handled?

Reply:
Correct. Without the `await`, it's almost like it will queue a background job, which won't be handled by the `try/catch` here

Reply:
That's not the case @sajidanower23 , it will definitely be caught by the exception handler

Confirming with example in REPL....

![image](https://github.com/user-attachments/assets/c78b96d0-d311-44f9-8e69-232f1bbedc0a)

Reply:
Doesn't look like it got caught by the exception handler - it did print "will not see this!"

Reply:
Also, note the "Uncaught Error" - it didn't get caught, right? 😄

Reply:
HAHAHA i confidently proved myself wrong 🤦

https://www.reddit.com/r/ConfidentlyWrong/

Reply:
Happens to the best of us

Reply:
I'm just happy to be correct, this is a seriously good piece of knowledge and I need to go check some code..... like.... quite a bit of code!